### DISCUSSION TITLE
Plotnine plot on specific ax

### DISCUSSION THREAD

**Discussion by aleixalcacer** [has2k1/plotnine#745] [Q&A]
Hi, I was wondering if it is possible to draw a graphic on a specific ax. For example, I would like to be able to do something like:

```python
...
plot0 = (ggplot(df, aes(x='x', y='y')) + geom_point())
plot1 = (ggplot(df, aes(x='y', y='z')) + geom_point())

fig, axs = plt.subplots(1, 2, figsize=(10, 5))
plot0.draw(axs[0])
plot1.draw(axs[1])
...
```

Thank you very much in advance, Aleix :)

**Comment by has2k1** [comment]
It is not possible. I realise the intention is to combine independent plots in a one figure. That will be implemented although not very soon.

**Reply by aleixalcacer** [reply]
Thank you for your sincere response :)
